homeassistant/components/alexa/smart_home.py

```diff
@@ -34,6 +36,19 @@
     TEMP_CELSIUS: 'CELSIUS',
 }
 
+API_THERMOSTAT_MODES = {
+    climate.STATE_HEAT: 'HEAT',
+    climate.STATE_COOL: 'COOL',
+    climate.STATE_AUTO: 'AUTO',
+    climate.STATE_ECO: 'ECO',
+    STATE_ON: 'AUTO',
```

Comment:
There is no `STATE_ON` or `STATE_OFF`

Reply:
daikin, eq3btsmart, generic_thermostat, hive, mqtt, nest, and radiotherm all appear to use `STATE_OFF` (sometimes in addition to `STATE_IDLE`). tesla uses `STATE_ON` instead of `STATE_AUTO`.
(My own thermostat uses the radiotherm platform).

Reply:
Looks like you're right.

I'm looking at the climate component right now and just the implementation of the state property is a mess.